pandokia/gen_expected.py

```diff
@@ -15,6 +15,7 @@
 #       fills in missing tests in test_run_to_check
 #
 #
+from __future__ import print_function
```

Comment:
Do we need this import statement in the other files with modified print statements too?

Reply:
Only in the ones where I didn't use the `%` syntax.  But I did put it there.

Reply:
Put another way, when you do `print('abc',2,4,6)` in Py2 it will print a tuple unless you add the future import.  But if you don't use that syntax, it works in both py2 and py3 without the future import.